JPS_Ontology/ontology/ontodevice/OntoDevice.csv

```diff
@@ -140,18 +140,18 @@ OperatingProperty,Class,,,,,,A property of an operating parameter of a device (e
-ActivityType,Class,,,,,,Represents the type of activity being performed (e.g., walking, biking).,https://www.theworldavatar.com/kg/ontodevice/,Activity Type
-OnFoot,Class,ActivityType,IS-A,,,,The device is on a user who is walking or running.,http://www.theworldavatar.com/kg/ontodevice/,On Foot
-InVehicle,Class,ActivityType,IS-A,,,,The device is in a vehicle.,http://www.theworldavatar.com/kg/ontodevice/,In Vehicle
-OnBicycle,Class,ActivityType,IS-A,,,,The device is on a bicycle.,http://www.theworldavatar.com/kg/ontodevice/,On Bicycle
+ActivityType,Class,,,,,,Represents the type of activity being performed (e.g.\, walking\, biking).,https://www.theworldavatar.com/kg/ontodevice/,Activity type
+OnFoot,Class,ActivityType,IS-A,,,,The device is on a user who is walking or running.,http://www.theworldavatar.com/kg/ontodevice/,On foot
+InVehicle,Class,ActivityType,IS-A,,,,The device is in a vehicle.,http://www.theworldavatar.com/kg/ontodevice/,In vehicle
+OnBicycle,Class,ActivityType,IS-A,,,,The device is on a bicycle.,http://www.theworldavatar.com/kg/ontodevice/,On bicycle
 Running,Class,OnFoot,IS-A,,,,A sub-activity of on foot where the user is running.,http://www.theworldavatar.com/kg/ontodevice/,Running
 Walking,Class,OnFoot,IS-A,,,,A sub-activity of on foot where the user is walking.,http://www.theworldavatar.com/kg/ontodevice/,Walking
-ConfidenceLevel,Class,,,,,,Represents the confidence level indicating the likelihood of the activity being accurate.,https://www.theworldavatar.com/kg/ontodevice/,Confidence Level
+Confidence level,Class,,,,,,Represents the confidence level indicating the likelihood of the activity being accurate.,https://www.theworldavatar.com/kg/ontodevice/,Confidence Level
 hasActivityType,Object Property,,Activity,ActivityType,,only,Links an activity to its corresponding type (e.g., walking, running).,https://www.theworldavatar.com/kg/ontodevice/,has activity type
 hasConfidenceLevel,Object Property,,Activity,ConfidenceLevel,,only,Links an activity to its confidence level.,https://www.theworldavatar.com/kg/ontodevice/,has confidence level
-detectedByDevice, Object Property,,prov:Activity, Smartphone,,only,Relates an activity with the device that detected it.,https://www.w3.org/ns/prov#,was associated with
-hasValue,Data Property,,ConfidenceLevel,xsd:int,,only,Represents the numerical confidence value.,https://www.theworldavatar.com/kg/ontodevice/,has value
-hasValue,Data Property,,ActivityType,xsd:string,,only,Represents the type of activity as a string.,https://www.theworldavatar.com/kg/ontodevice/,has value
+detectedByDevice, Object Property,,prov:Activity, Device,,only,Relates an activity with the device that detected it.,https://www.theworldavatar.com/kg/ontodevice/,detected by device
+hasValue,Data Property,,ConfidenceLevel,Integer,,only,Represents the numerical confidence value.,https://www.theworldavatar.com/kg/ontodevice/,has value
```

Comment:
Integer and String for hasValue seems to be in the wrong columns, please refer to hasID data property.

hasID,Data Property,,,Device,String,,A relationship that describes the ID assigned to a device.,https://www.theworldavatar.com/kg/ontodevice/,has ID